src/Workspaces/Core/Portable/Shared/Collections/IntervalTree`1.cs

```diff
@@ -17,6 +17,7 @@ internal partial class IntervalTree<T> : IEnumerable<T>
         public static readonly IntervalTree<T> Empty = new IntervalTree<T>();
 
         protected Node root;
+        private readonly int count;
```

Comment:
Or just #if DEBUG it, since that way it'll be clear it's for, well, debugging. :-)